docs/data/routes/docs/techniques/ssr/sitecore-experience-edge/en.md

```diff
@@ -0,0 +1,74 @@
+---
+name: ssr-experience-edge
+routeTemplate: ./data/component-templates/article.yml
+title: SSR using Sitecore Experience Edge
+---
+
+# Server-side rendering using Sitecore Experience Edge
+
+You can configure server-side rendering (SSR) for your JSS application using a Sitecore Experience Edge endpoint. 
+
+Sitecore provides a [sample for server-side rendering using a Sitecore Experience Edge endpoint](https://github.com/Sitecore/jss/tree/dev/samples/node-headless-ssr-experience-edge), Node.js, and Express. It renders your JSS application, querying layout and dictionary data, without proxying the requests.
+
+> The sample for SSR using a Sitecore Experience Edge endpoint is experimental and not officially supported by Sitecore.
+
+## Pre-requisites
+
+To use this technique you need:
+
+1. A Sitecore instance with [Headless Services installed](/docs/client-frameworks/getting-started/jss-server-install).
+
+2. A Sitecore API key. 
+
+3. A React, Vue or Angular application built with JSS that:
+
+   - Supports [server-side rendering in integrated mode](/docs/techniques/ssr/integrated-mode-ssr).
+
+   - Supports the Experience Edge GraphQL schema. 
+
+   > You will need to update your application's GraphQL components and queries to be compatible with the Experience Edge schema. See the [GraphQL-ConnectedDemo](https://github.com/Sitecore/jss/blob/master/samples/nextjs/src/components/graphql/GraphQL-ConnectedDemo.tsx) component and [the associated GraphQL query](https://github.com/Sitecore/jss/blob/dev/samples/nextjs/src/components/graphql/GraphQL-ConnectedDemo.graphql) in the Next.js sample application.
```

Comment:
@OlegJytnik - can you confirm that this statement is accurate? I thought that Headless Services GraphQL endpoint and Edge endpoint are supposed to function that same

Reply:
This is speaking about the legacy/non-edge GraphQL schema which all of the other sample apps still use. Maybe we need to call that out specifically to be clear?